chpl/chpl-resources/src/main/resources/system-triggers.xml

```diff
@@ -5,6 +5,16 @@
     xsi:schemaLocation="http://www.quartz-scheduler.org/xml/JobSchedulingData http://www.quartz-scheduler.org/xml/job_scheduling_data_2_0.xsd"
     version="2.0">
     <schedule>
+        <trigger>
+            <cron>
+                <name>generateUpdateListingStatusReport</name>
+                <group>generateUpdateListingStatusReportTrigger</group>
+                <job-name>updatedListingStatusReportCreatorJob</job-name>
+                <job-group>systemJobs</job-group>
+                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
+                <cron-expression>0 5 9 * * ?</cron-expression> <!-- At 0505 UTC every day -->
```

Comment:
The comment says 0505, but the cron job is, either 0509 or 0905. I can't recall which